NOTICE.txt

```diff
@@ -43,21 +1390,33 @@ SOFTWARE.
 
 ---
 
-## bootstrap
+## jq
 
-This product contains 'bootstrap' by The Bootstrap Authors.
+This product contains 'jq' by Stephen Dolan.
 
-The most popular front-end framework for developing responsive, mobile first projects on the web.
+jq is a lightweight and flexible command-line JSON processor.
 
 * HOMEPAGE:
-  * https://getbootstrap.com/
+  * https://github.com/stedolan/jq
+
+* LICENSE:
+
```

Comment:
If this is the correct one, then we need to contact the repo owners because there is no license listed on their repo.

Reply:
I believe it is MIT licensed.
its license is not in the usual place but in [copying](https://github.com/stedolan/jq/blob/master/COPYING)